src/data/players.ts

```diff
@@ -32,10 +35,43 @@ export const players: Player[] = [
 	}
 ];
 
-export const playersById = players.reduce<{ [key: string]: Player }>(
-	(acc, curr) => ({
-		...acc,
-		[curr.id]: curr
-	}),
-	{}
+function createPlayersStore() {
+	const { subscribe, update: storeUpdate } = writable<Player[]>(initialPlayers);
```

Comment:
Since Players is now something that is editable in the app, I updated `players` to be a [writable store](https://svelte.dev/docs#run-time-svelte-store) instead of just a play array. This is a little new to me, but I basically created a custom store that exposes some simpler functions (`add` and `update`) for manipulating this data store.